examples/nlp/language_modeling/tuning/conf/megatron_gpt_sft.yaml

```diff
@@ -115,22 +116,23 @@ model:
       num_workers: 4
       memmap_workers: ${model.data.train_ds.memmap_workers}
       pin_memory: True
-      max_seq_length: 2048
+      max_seq_length: ${model.data.train_ds.max_seq_length}
       min_seq_length: 1
       drop_last: False
       context_key: ${model.data.train_ds.context_key}
       label_key: ${model.data.train_ds.label_key}
-      add_eos: ${model.data.train_ds.add_eos}
+      add_eos: False
```

Comment:
Shouldn't we need this to be True for our old non-chat SFT models?

Reply:
@hsiehjackson why do we want to hard-code false? seems like we can leave it as is.

Reply:
Do we need `eos` token during validation? For generation, I don't think we need to add this, so I remove it. But for calculating loss, I think we may need this to compare training loss. Which option do you think is reasonable?